src/systems/systems/src/hydraulic/update_iterator.rs

```diff
@@ -71,50 +71,44 @@ impl Iterator for FixedStepLoop {
 /// Provides maximum fixed time interval looping.
 ///
 /// ## Example scenario
-/// With a max fixed time interval of 10 ms and a frame delta of 35 ms, this type will provide three
-/// iterations of 10 ms, and one iteration of 5ms to complete the 35ms total delta.
+/// With a max fixed time interval of 10 ms and a frame delta of 35 ms, this type will provide four
+/// iterations of 8.75ms, thus completing the 35ms total delta.
 #[derive(Copy, Clone)]
 pub struct MaxFixedStepLoop {
```

Comment:
**thought**: I thought `Fixed` meant the time for most loops was equal. What did and does `Fixed` refer to if not that?

Reply:
Yeah it's not Fixed it's MaxFixed: it's the Max that is fixed... Maybe MaxStepLoop is enough as a name then?